pybossa/core.py

```diff
@@ -283,6 +282,18 @@ def setup_social_networks(app):
         print inst
```

Comment:
yes. I will also leave the print as it is quite useful for local PyBossa installations, what do you think?